Makefile

```diff
@@ -34,3 +34,8 @@ generate:
 test:
 	go test -v ./cmd/... ./pkg/...
 
+test/build:
```

Comment:
These are phony targets and need to be marked as such.

Separately: nit-wise, I'm not a huge fan of the name of this -- maybe `test-project-generation`?

Reply:
Done 👍